src/index.tsx

```diff
@@ -124,6 +124,10 @@ startMathJax();
 // load optimize
 loadOptimize(window, app.store);
 
+function cookiesBlocked(e: DOMException) {
+  return ['SecurityError', 'NotSupportedError'].includes(e.name);
```

Comment:
you're accessing e.name here assuming e is a DOMException and its not complaining because the e coming into the catch on line 148 is `any`, but you observed some errors coming into there that were not subclass of `Error`, i think you saw `undefined` coming in ? anyway you probably want some additional checks here to prevent errors accessing props of nonobjects or whatever the error is.

if you're gonna leave this function with the `DOMException` type, maybe change the condition below like

```diff
-    if (cookiesBlocked(e)) {
+    if (e instanceof DOMException  && cookiesBlocked(e)) {
```

or you could change this function here to accept `unknown` or something for that type and do the `instanceof` check here

Reply:
Yes, it had caught `undefined` when running locally. I have it check for `Error` first (so it won't try to report undefined), and for `DOMException` in `cookiesBlocked`
The serviceWorker still reports an error in the console, but it is not an uncaught error.